docs/v1/P4Runtime-Spec.mdk

```diff
@@ -3010,6 +3018,14 @@ the `TableEntry` entity, which has the following fields:
   implement idle-timeout support for the table, if applicable. See
   [Idle-timeout](#sec-idle-timeout) section for more information.
 
+* `resources`, a `repeated GenericTableEntry` to contain a list of
+  target-specific direct resources. For example, if a target supports special
+  direct resources like x-meter and y-counter, then the p4info will contain
+  GenericTable representations of the same and the list here can contain one
+  member for each of the direct resources. If the generic match fields are set
+  in these entries, it is an error. The IDs will be mentioned in the
```

Comment:
Most other places in the spec I believe say something like "If any match fields are set in these `GenericTableEntry` messages, the server must return a InvalidSomethingType error.", i.e. they specify the particular error status the server must return in this situation.